ci/buildkite.yml

```diff
@@ -2,6 +2,11 @@
 #
 # Release tags use buildkite-release.yml instead
 steps:
+  - command: "true"
```

Comment:
@ryoqun - here's where we could run a dependabot specific ci script to fix up the lock files

Reply:
Oh this look nice!